### PR TITLE
Add kpod import command

### PR DESCRIPTION
**Pull request by umohnani8**
Imports a tarball to create a filesystem image
A one word change to the error message in kpod history.
Vendor in latest containers/image

Signed-off-by: umohnani8 <umohnani@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
@rhatdan PTAL

**Comment by TomSweeneyRedHat** [comment]
1.8 failed.  I think a memory hiccup, but I'm not sure.

**Comment by rhatdan** [comment]
@umohnani8 Could you move this to the other repo.

**Comment by umohnani8** [comment]
@rhatdan moving

File: docs/kpod-import.1.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+## OPTIONS
+
+**--change, -c**
+Apply imgspecv1 configurations to the created image
+Possible configurations include:
+**USER** | **EXPOSE** | **ENV** | **ENTRYPOINT** | **CMD** | **VOLUME** | **WORKDIR** | **LABEL** | **STOPSIGNAL**
+
+**--message, -m**
+Set commit message for image imported
```
Comment (TomSweeneyRedHat) [inline docs/kpod-import.1.md]: nit "for the image imported"  add 'the'

File: docs/kpod-import.1.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+Copying blob sha256:b41deda5a2feb1f03a5c1bb38c598cbc12c9ccd675f438edc6acd815f7585b86
+ 25.80 MB / 25.80 MB [======================================================] 0s
+Copying config sha256:d61387b4d5edf65edee5353e2340783703074ffeaaac529cde97a8357eea7645
+ 378 B / 378 B [============================================================] 0s
+Writing manifest to image destination
+Storing signatures
+```
+
+```
+kpod import http://lacrosse.redhat.com/~umohnani/ctr.tar url-image
```
Comment (TomSweeneyRedHat) [inline docs/kpod-import.1.md]: might want to consider changing the http address to something fake.  Perhaps:
http://myhost.mycompany.com/~myusername/ctr.tar

Just so yours won't get blasted.  Ditto next line.

File: docs/kpod-import.1.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+**kpod import [GLOBAL OPTIONS]**
+
+**kpod import [OPTIONS] CONTAINER**
+
+## OPTIONS
+
+**--change, -c**
+Apply imgspecv1 configurations to the created image
+Possible configurations include:
+**USER** | **EXPOSE** | **ENV** | **ENTRYPOINT** | **CMD** | **VOLUME** | **WORKDIR** | **LABEL** | **STOPSIGNAL**
```
Comment (TomSweeneyRedHat) [inline docs/kpod-import.1.md]: Can we alphabetize these?

File: completions/bash/kpod [hunk 4]
```diff
@@ -711,6 +711,28 @@ _kpod_history() {
...
 }
 
+
+_kpod_import() {
+    local options_with_args="
+     --change
+     -c
+     --message
+     -m
+     "
```
Comment (TomSweeneyRedHat) [inline completions/bash/kpod]: do you need to add help and h here too?

File: docs/kpod-import.1.md (hunk 1 above)
Comment (umohnani8) [reply docs/kpod-import.1.md]: fixed

File: docs/kpod-import.1.md (hunk 2 above)
Comment (umohnani8) [reply docs/kpod-import.1.md]: fixed, thanks :)

File: docs/kpod-import.1.md (hunk 3 above)
Comment (umohnani8) [reply docs/kpod-import.1.md]: fixed

File: completions/bash/kpod (hunk 4 above)
Comment (umohnani8) [reply completions/bash/kpod]: fixed

**Comment by umohnani8** [comment]
Opened in new repo https://github.com/projectatomic/libpod/pull/12
All reviews fixed in that.

**Pull request closed without merging**